### DISCUSSION TITLE
Vue/CLI no me permite compilar

### DISCUSSION THREAD

**Discussion by SamaelGT** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#545] [Q&A] [answered]
![Captura de Pantalla 2022-03-27 a la(s) 16 06 57](https://user-images.githubusercontent.com/99088608/160304933-ed8cd301-8d37-431b-9983-28559cb929ba.png)

Hola, intento realizar la practica de Vue pero al momento de crearla me sale que tengo que actualizarla, y al momento de actualizarla y volver a intentar me sale lo mismo, ya intente reiniicando el ordenador, cambiando el directorio, pero aun asi no puedo realizarla, alguien que me pueda ayudar por favor  😖

![Captura de Pantalla 2022-03-27 a la(s) 16 06 11](https://user-images.githubusercontent.com/99088608/160305042-636c2d79-984d-4ab6-b079-bb395415a5e5.png)

**Comment by carlos0318** [comment, marked as answer]
Puedes utilizar este comando "npm init vue@latest" para crear un nuevo proyecto con la ultima versión de vue.

**Reply by SamaelGT** [reply]
ty, lo intentare

**Reply by SamaelGT** [reply]
![Captura de Pantalla 2022-03-29 a la(s) 14 44 39](https://user-images.githubusercontent.com/99088608/160704090-87441367-7a9e-4f4a-9ea9-4d31699e8631.png)
Si me permitio crearlo, pero ahora no me permite correr el server

**Reply by carlos0318** [reply]
Utiliza primero "npm install" para instalar todo y después  "npm run dev" para correr el proyecto.

**Comment by hackit92** [comment]
Por lo que veo se te está instalando la versión de Vue 2 y por ende solo tienes acceso a comandos de la misma, que versión de nodejs instalaste?

**Reply by SamaelGT** [reply]
![Captura de Pantalla 2022-03-28 a la(s) 14 28 51](https://user-images.githubusercontent.com/99088608/160481648-bbb45ce9-523a-4d5e-bdcc-c35609e83811.png)
Esta fue la version de node que instale y ya intente actualizando el vue en mi ordenador pero aun asi sigue mostrandome que es la version 2.9.6 ya intente hasta instalar con sudo pero aun asi me marca que es la version esa

**Reply by hackit92** [reply]
Yo tuve un problemilla al tratar de ir a la última versión y tuve que desinstalar primero y luego hacer todo el proceso aunque en mi caso me fui por la 17.8.0 desde el principio, creo que reinstalar podría ser la solución.

**Reply by SamaelGT** [reply]
Buenas tardes, ya intente desinstalando y volviendo a instalar y aun no me permite, pero utilice el comando del comentario anterior y es el que me permitio realizarlo. Gracias 👍🏽

**Reply by JuanSalvador73** [reply]
Buenas Explorers, yo tengo el siguiente error en Windows 10:

![error_vue](https://user-images.githubusercontent.com/99113565/160967567-365bfa4f-984e-4bb8-ba5b-f39d3e6d2223.png)

Según entiendo, los pasos a seguir para crear el proyecto son los siguientes:

1.- Instalar Node ( por si no lo tenemos ) - lo instalé en disco D, no C, no creo eso sea importante -
2.- Instalar VUE en CDM (línea de comando)
3.- Abrir Visual Studio Code, y usando la terminal crear el proyecto: vue create mi-proyecto

¿Alguien me puede decir si son los pasos correctos, omití algo? Ya en otro post me recomendaron  usar este:
npm init vue@latest

Pero ni así. Y veo que no soy el único con ese problema, con la práctica CND de los perfiles me salió bien...
